### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

Partially fixing https://github.com/pandas-dev/pandas/issues/53668

### CODE REVIEW

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: pandas/_libs/tslibs/timestamps.pyx [hunk 1]
```diff
@@ -1531,6 +1540,99 @@ class Timestamp(_Timestamp):
...
+        >>> ts = pd.Timestamp('2023-01-01 10:00:00', tz='Europe/Brussels')
+        >>> ts
+        Timestamp('2023-01-01 10:00:00+0100', tz='Europe/Brussels')
+        >>> ts.timetz()
+        datetime.time(10, 0, tzinfo=<DstTzInfo 'Europe/Brussels' CET+1:00:00 STD>)
+        """
+        try:
+            _dt = datetime(self.year, self.month, self.day,
+                           self.hour, self.minute, self.second,
+                           self.microsecond, self.tzinfo, fold=self.fold)
```
[the remark this answers is not in the document]
Comment (DeaMariaLeon) [reply pandas/_libs/tslibs/timestamps.pyx:1605]: I must be missing something but I get: `AttributeError: 'datetime.time' object has no attribute 'timetz'`
I checked the docs too. So, isn't timetz a datetime method?

File: pandas/_libs/tslibs/timestamps.pyx [hunk 2]
```diff
@@ -1531,6 +1542,85 @@ class Timestamp(_Timestamp):
...
+        >>> ts = pd.Timestamp('2023-01-01 10:00:00', tz='Europe/Brussels')
+        >>> ts
+        Timestamp('2023-01-01 10:00:00+0100', tz='Europe/Brussels')
+        >>> ts.timetz()
+        datetime.time(10, 0, tzinfo=<DstTzInfo 'Europe/Brussels' CET+1:00:00 STD>)
+        """
+        _dt = dt.time(self.hour, self.minute, self.second,
+                      self.microsecond, self.tzinfo)
+        return _dt
+
```
[the remark this answers is not in the document]
Comment (MarcoGorelli) [reply pandas/_libs/tslibs/timestamps.pyx:1600]: I don't think there is a `super().timetz` method

[the remark this answers is not in the document]
Comment (MarcoGorelli) [reply pandas/_libs/tslibs/timestamps.pyx:1600]: oh wait, sorry, there is, that's why there was no definition previously

in which case, yeah, if it doesn't give incorrect results for negative dates or anything, then that looks fine

**Review by MarcoGorelli — APPROVED** [review]
thanks @DeaMariaLeon !

**Pull request merged**